config/jobs/kubernetes/sig-k8s-infra/trusted/releng/releng-trusted.yaml

```diff
@@ -30,7 +30,7 @@ postsubmits:
     decorate: true
     decoration_config:
       timeout: 4h
-    run_if_changed: 'k8s.gcr.io/((images/.*/images\.yaml)|(manifests/.*/promoter-manifest\.yaml))'
+    run_if_changed: 'registry.k8s.io/((images/.*/images\.yaml)|(manifests/.*/promoter-manifest\.yaml))'
```

Comment:
Can you revert all the changes to this file please? This will break the image promotion jobs right now. I will resolve this as part of the image registry freeze in March.